### PR TITLE
support horonomic/drone

### PR DESCRIPTION
**Pull request by k-okada**
- add README.md from #3
- support sending linear.z
- support horonomic base, send commmand with keyboard down
- add dummy linear.y

### CODE REVIEW

File: teleop_twist_keyboard.py [hunk 1]
```diff
@@ -19,18 +19,30 @@
 e/c : increase/decrease only angular speed by 10%
 anything else : stop
 
+For Horonomic base, holding down Shift key
+
+For Drones, use t/b to control elevation
```
Comment (trainman419) [inline teleop_twist_keyboard.py]: These instructions are pretty vague; they still leave open questions about which keys are active when holding shift, and how `t` and `b` control elevation.

Comment (k-okada) [inline teleop_twist_keyboard.py]: t is up, increase z value and b for another, how about
`for Drones, use t for up and b for down`
??

--
◉ Kei Okada

Comment (trainman419) [inline teleop_twist_keyboard.py]: I'd like to propose:

> For Holonomic mode (strafing), hold down the shift key:
> ```
 U  I  O
 J  K  L
 M  <  >
```
> 
> For vertical movement, use `t` for up and `b` for down, at the same speed.
>
> Holonomic mode and vertical mode both use the linear speed setting.

For properly implementing Holonomic mode, this also needs bindings for `I` and `<`

**Comment by trainman419** [comment]
Note also that the accepted spelling is "Holonomic"

**Comment by k-okada** [comment]
ok, updated the PR

**Comment by trainman419** [comment]
Looks reasonable. Please update the README too.

**Comment by k-okada** [comment]
ok, done

--
◉ Kei Okada

**Comment by trainman419** [comment]
Thanks!

**Comment by trainman419** [comment]
Released into Indigo, Jade and Kinectic as version 0.6.0

**Comment by k-okada** [comment]
Thank you!

**Pull request merged**